packages/widgetbook_addon/lib/src/widgetbook_addon.dart

```diff
@@ -52,9 +52,12 @@ abstract class WidgetbookAddOn<T extends WidgetbookAddOnModel<T>> {
     );
   }
 
-  Widget build(
-    BuildContext context,
-  );
+  Widget buildSetting(BuildContext context);
+
+  /// Wraps use cases with a custom widget depending on the addon [setting].
+  Widget buildUseCase(BuildContext context, Widget child) {
```

Comment:
@jenshor I think `buildUseCase` is a good name, and it goes well with what it does, and follows the same naming convention of `buildSetting`. What do you think?